src/Files.Package/Package.appxmanifest

```diff
@@ -9,7 +9,7 @@
          xmlns:uap4="http://schemas.microsoft.com/appx/manifest/uap/windows10/4"
          xmlns:uap5="http://schemas.microsoft.com/appx/manifest/uap/windows10/5"
          IgnorableNamespaces="uap uap5 mp rescap desktop4 desktop">
-    <Identity Name="FilesDev" Publisher="CN=53EC4384-7F5B-4CF6-8C23-513FFE9D1AB7" Version="2.0.42.0" />
+    <Identity Name="FilesDev" Publisher="CN=Files" Version="2.0.42.0" />
```

Comment:
What's the reason of this change?

Reply:
The real reason is that the installation of Files on my device somehow got into a bad state and while debugging I found that changing this fixed the issue. I can revert it if you want.